src/components/tag-filter/__tests__/tag-filter.test.tsx

```diff
@@ -0,0 +1,188 @@
+import { render, screen, userEvent } from '@/test-utils/rtl';
+
+import TagFilter from '../tag-filter';
+import { type TagFilterOptionConfig } from '../tag-filter.types';
+
+jest.mock('../selectable-tag/selectable-tag', () =>
+  jest.fn(({ value, onClick, children, startEnhancer }) => (
+    <button
+      data-testid={`selectable-tag-${children}`}
+      data-selected={value}
+      onClick={onClick}
+    >
+      {startEnhancer && startEnhancer()}
+      {children}
+    </button>
+  ))
+);
```

Comment:
Can the unit tests be achieved without jest.mocking selectable-tag?

Reply:
Should be possible. Normally, we mock out all dependencies in our unit tests, but I don't feel too strongly about that.